src/app/global/utils.nim

```diff
@@ -155,7 +155,7 @@ QtObject:
       return false
 
   proc isChatKey*(self: Utils, value: string): bool {.slot.} =
-      result = (conversion.startsWith0x(value) and conversion.isHexFormat(value) and len(value) == 132) or self.isCompressedPubKey(value)
+    result = (conversion.startsWith0x(value) and len(value) == 132) or self.isCompressedPubKey(value)
```

Comment:
That is the call that generated a ton of logs. Just a red herring, but to be honest, that call made no sense (since it's made to check addresses) and it's useless

Reply:
hmm don't you want to check the characters are valid hex ones before returning true? I'm guessing that was the intention of the one you removed

Reply:
That was the intention, yes, but it probably never worked, because we try to convert it to an Address, which chat keys are **not**
```nim
proc decodeHexAddress*(strAddress: string): Address =
  var hexAddressValue: Address
  try:
    hexAddressValue = fromHex(Address, strAddress)
    ...

proc isHexFormat*(strAddress: string): bool =
  try:
    discard decodeHexAddress(strAddress)
    ...
```

IMO the remaining checks are sufficient for the use case. But if we want, we can add a new `isHex` function, because the current one is more a `isAddress`

Reply:
yeah yeah, I get that `isHexFormat ` was giving us false negatives, but now we'd get false positives instead (i.e. a string like `0x!@#ˆ&!ˆ@$*&(ˆ!*&@$ˆ(*&ˆ...` that's 132 chars long)

Reply:
Yeah, I'll add a basic regex then to check